crates/sui-indexer-alt-jsonrpc/src/lib.rs

```diff
@@ -71,6 +76,9 @@ pub struct RpcService {
 
     /// Cancellation token controlling all services.
     cancel: CancellationToken,
+
+    /// Configuration for the RPC service
+    config: RpcConfig,
```

Comment:
The `RpcConfig` is intentionally not included in `RpcService` -- if you need it here, it's a sign that there's some kind of mismatch going on. `RpcService` is agnostic to the exact schema of the RPC being served, while `RpcConfig` is specific to our current schema.